### PR TITLE
Headless portal

### PR DESCRIPTION
**Pull request by TheGeekyUnicorn**

### CODE REVIEW

File: Umbraco-Headless/Getting-Started-Cloud/The-Headless-Portal/index.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# The Umbraco Heartcore Portal
+
```
Comment (sofietoft) [inline Umbraco-Headless/Getting-Started-Cloud/The-Headless-Portal/index.md]: Should we add some intro text here, explaining what kind of information can be found in the article?

File: Umbraco-Headless/Getting-Started-Cloud/The-Headless-Portal/index.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+After you have logged into the Cloud Portal you are presented with the project overview.
```
Comment (sofietoft) [inline Umbraco-Headless/Getting-Started-Cloud/The-Headless-Portal/index.md]: Would be great to have a link to the Portal from here!

File: Umbraco-Headless/Getting-Started-Cloud/The-Headless-Portal/index.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+From here you can see a list of all your projects. If you have many projects, you are able to search for specific projects from the search bar in the top right corner.
+
+Clicking on a project will take you to the project where you will get an overview of your environment(s).
+
+![Portal Project Overview](images/projectOverview.png)
+
+## Manage Environments
+
+From here you are able to create multiple environment so you might have a Development environment as well. Depending on what plan you have chosen, you will have acces to one or more environment on your project.
```
Comment (sofietoft) [inline Umbraco-Headless/Getting-Started-Cloud/The-Headless-Portal/index.md]:
```suggestion
From here you are able to create multiple environments so you might have a Development environment as well. Depending on what plan you have chosen, you will have acces to one or more environment on your project.
```

File: Umbraco-Headless/Getting-Started-Cloud/The-Headless-Portal/index.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+![Manage the environments](images/manageEnvironments.png)
+
+If you want to add an environment all you have to do is click the big green button. Umbraco will create the environment for you.
+
+When you are done you can go a head and click on the Done, take me back link in the top-right corner to get back to the project overview.
```
Comment (sofietoft) [inline Umbraco-Headless/Getting-Started-Cloud/The-Headless-Portal/index.md]:
```suggestion
When you are done you can go ahead and click on 'Done, take me back' in the top-right corner to get back to the project overview.
```

File: Umbraco-Headless/Getting-Started-Cloud/The-Headless-Portal/index.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+It is also here you are able to add the Technical Contact information such as Name, Email and a telephone number.
+
+![Edit Team Members page](images/editTeam.png)
+
+### Webhooks
+
+Here you are able to set up a webhook that will gather all the information related to deployments. This information is very useful if you har set up something like Slack integration so every time a deployment has been made a message will be posted.
+
+The webhooks that can be added here is not the same as the webhooks that can be created from the Backoffice.
```
Comment (sofietoft) [inline Umbraco-Headless/Getting-Started-Cloud/The-Headless-Portal/index.md]: Would be cool if this was added like a note or tip instead, right?

Check out this article to learn more about how to format notes and tips: https://our.umbraco.com/documentation/Contribute/Markdown-Conventions/#adding-notes-warnings-tips

File: Umbraco-Headless/Getting-Started-Cloud/The-Headless-Portal/index.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+### Upgrade plan
+
+Here you will be able to upgrade your trial to a plan that fits your needs.
+
+### Rename project
+
+Here you are able to rename your project. All default umbraco.io hostnames will be updated to match the new name. Also note that all clone urls will be updated to match the new projectname. If you have local git repositories remember to update the urls.
```
Comment (sofietoft) [inline Umbraco-Headless/Getting-Started-Cloud/The-Headless-Portal/index.md]: I don't think we should mention "cloning down" and "local repositories" here, as that will not be an option for Heartcore. What do you think?

```suggestion
Here you are able to rename your project. All default umbraco.io hostnames will be updated to match the new name. Also note that all clone URLs will be updated to match the new projectname. If you have local git repositories remember to update the URLs.
```

File: Umbraco-Headless/Getting-Started-Cloud/The-Headless-Portal/index.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+
+### Delete project
+
+From this page you can delete your project. When deleting the project your subscription will automatically be cancelled as well. Deleting the project is a **permanent action**.
+
+![Delete project](images/deleteProject.png)
+
+## Go to backoffice
+
+Clicking on the Go to backoffice link will open up a new tab with the login screen for the Backoffice. If you would like to learn more, you can read our documentation for the [Heartcore Backoffice](../The-Headless-Portal).
```
Comment (sofietoft) [inline Umbraco-Headless/Getting-Started-Cloud/The-Headless-Portal/index.md]:
```suggestion
Clicking on the 'Go to backoffice' link will open up a new tab with the login screen for the Backoffice. If you would like to learn more, you can read our documentation for the [Heartcore Backoffice](../The-Headless-Portal).
```

File: Umbraco-Headless/Getting-Started-Cloud/The-Headless-Portal/index.md (hunk 6 above)
Comment (rasmusjp) [reply Umbraco-Headless/Getting-Started-Cloud/The-Headless-Portal/index.md]: Maybe instead we could mention that you'll need to update the `Umb-Project-Alias` header in the requests to the APIs?

File: Umbraco-Headless/Getting-Started-Cloud/The-Headless-Portal/index.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+The webhooks that can be added here is **not the same** as the webhooks that can be created from the Backoffice.
+:::
+
+### Upgrade plan
+
+Here you will be able to upgrade your trial to a plan that fits your needs.
+
+### Rename project
+
+Here you are able to rename your project. All default umbraco.io hostnames will be updated to match the new name. When renaming a project you will also need to update the `Umb-Project-Alias` header when you are sending requests to the API.
```
Comment (sofietoft) [inline Umbraco-Headless/Getting-Started-Cloud/The-Headless-Portal/index.md]:
```suggestion
Here you are able to rename your project. All default umbraco.io hostnames will be updated to match the new name. When renaming a project you will also need to update the `Umb-Project-Alias` header when you are sending requests to the API, as changing the name of the project will also change the project alias.
```

**Pull request merged**